docs/man_pages/project/configuration/web-view-set.md

```diff
@@ -0,0 +1,42 @@
+web-view set
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder web-view set <Platform> <WebViewName>`
+
+Sets the selected web view for the current project and updates the integrated plugins to match it.
+
+<% if(isConsole)  { %>
+<% if(isNativeScript)  { %>
+WARNING: This command is not applicable to NativeScript projects. To view the complete help for this command, run `$ appbuilder help web-view set`
+<% } %>
+<% if(isMobileWebsite)  { %>
+WARNING: This command is not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help web-view set`
+<% } %>
+<% } %>
+<% if((isConsole && isCordova) || isHtml) { %>
+
+### Attributes
+* `<Platform>` is the target mobile platform for which you want to change your web view. 
+* `<WebViewName>` is the web view name as listed by `$ appbuilder web-view`.
+<% } %>
+<% if(isHtml) { %> 
+### Command Limitations
+
+* You cannot run this command on NativeScript projects.
+* You cannot run this command on mobile website projects.
+
```

Comment:
If indeed this is not runnable for Windows Phone, add the following line:

\* You cannot configure web views for Windows Phone.